src/application/config/routes.php

```diff
@@ -95,6 +95,6 @@
 $route['api/v2/providers']['put'] = 'api/v2/providersv2/updateProvider';
 $route['api/v2/services']['put'] = 'api/v2/servicesv2/updateService';
 $route['api/v2/appointments']['get'] = 'api/v2/appointmentsv2/getAllAppointmentByPeriodTime';
-$route['api/v2/total/appointments']['get'] = 'api/v2/appointmentsv2/getTotalAppointmentGroupByGender';
+$route['api/v2/statistic/appointments']['get'] = 'api/v2/appointmentsv2/getTotalAppointmentGroupByGender';
```

Comment:
=)))))) the model that we are querying is appointment so that the api have to be appointments/statistic. please change

Reply:
both statistic/appointments and appointments/statistic are logical but I prefer the later.

Reply:
updated to appointments/statistic